trajectory_msgs/JointTrajectoryPoint pose
uint8 inactiveRobots
trajectory_msgs/JointTrajectoryPoint inactiveRobotsPose
---
bool collision